css/pages/_content.scss

```diff
@@ -198,12 +212,18 @@
         font-size: 13px;
         font-weight: 600;
         background: $grey;
+        text-transform: capitalize;
+        text-transform: lowercase;
+
+        &::first-letter {
+            text-transform: capitalize;
+        }
```

Comment:
😱
```suggestion
        text-transform: capitalize;
```

Reply:
lowercase is required value because text uppercase in the md file